phing/tasks/setup.xml

```diff
@@ -202,7 +202,14 @@
 
   </target>
 
-  <target name="setup:config-import" description="Import configuration from the config directory.">
+  <target name="setup:pre-config" description="Setup before importing configuration.">
+    <echo>Pre config setup... </echo>
```

Comment:
This echo statement should not be necessary, as the hook invokation itself echoes similar information.